debian/install.sh

```diff
@@ -77,25 +174,29 @@ echo "3 - exim";
 echo "N - Do not install";
 echo;
 echo "Recommended: 1 (sendmail)"; echo;
-read -r -p "Install an MTA? [1] : " response
-
-if [[ $response =~ ^([nN][oO])$ ]]; then
-    # do not install
-    MTAOPTION=
-elif [ -z $response ]; then    
-	# sendmail default
-    MTAOPTION="sendmail sendmail-bin";
-elif [ $response == 1 ]; then    
-	# sendmail 
-    MTAOPTION="sendmail sendmail-bin";
-elif [ $response == 2 ]; then    
-	# sendmail
-    MTAOPTION="postfix";
-elif [ $response == 3 ]; then    
-	# sendmail
-    MTAOPTION="exim4-base";
+if [ -z "${arg_MTA+x}" ]; then
```

Comment:
Script does not prompt for MTA when run without parameters.  Logic here is inverted.  If this variable is empty it should prompt the user else read the variable.